custom/icds_reports/utils/aggregation_helpers/agg_awc.py

```diff
@@ -541,12 +541,16 @@ def _transform_column(column_tuple):
         columns = list(map(_transform_column, columns))
 
         group_by = ["state_id"]
+        child_location = 'district_is_test'
         if aggregation_level > 1:
             group_by.append("district_id")
+            child_location = 'block_is_test'
         if aggregation_level > 2:
             group_by.append("block_id")
+            child_location = 'supervisor_is_test'
         if aggregation_level > 3:
             group_by.append("supervisor_id")
+            child_location = 'awc_is_test'
```

Comment:
nit: this will reassign the `child_location` multiple times if `aggregation_level` is greater than 2, once for each level. if you reversed the order of the conditions it will only reassign once